docker/depends/pecan.depends.R

```diff
@@ -17,7 +17,10 @@ lapply(c(
 'r-lib/vdiffr@v1.0.2',
 'ropensci/geonames',
 'ropensci/nneo'
-), remotes::install_github, lib = rlib)
+), lib = rlib)
+
+#temporary debug
+print(gh::gh_token())
```

Comment:
@aariq Can you please delete all logs from this test as soon as you're finished?

Reply:
I can, but FYI, it doesn't print the whole token